test/visualTest.ts

```diff
@@ -354,5 +354,14 @@ module powerbi.extensibility.visual.test {
                 expect(context).not.toBeNull();
             });
         });
+
+        describe("Selection", () => {
+            it("Check index of the data-point after filtering", () => {
+                const item: { text: string, index: number } = VisualClass.converter(dataView, null, visualBuilder.visualHost, null)
```

Comment:
Why do you declare a type by creating one? I suppose that you might use the original type.